metadata_tools/hosts/GO_0xxx/gcp_index_startup.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+apt-get update -y
+
+apt-get install -y python3 python3-pip python3-venv git
+cd || exit 1
+git clone https://github.com/SETI/rms-metadata-tools.gitnew_overrides
+cd rms-metadata-tools || exit 1
+source venv/bin/activate || echo 'No virtual environment' ; exit 1
```

Shellcheck (0.11.0)

[info] Not following: venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)